src/main/app/utils/tacticalIdamRedirectHelper.ts

```diff
@@ -0,0 +1,28 @@
+import * as config from 'config'
+import * as express from 'express'
+import { buildURL } from 'utils/callbackBuilder'
+import { Paths, Paths as AppPaths } from 'app/paths'
```

Comment:
Are you sure you want to import Paths twice but once with alias?

Reply:
hah nope, intellij being too keen.